server/index.js

```diff
@@ -58,13 +57,41 @@ app.get('/products/:product_id/related', (req, res) => {
   getRelatedProducts(productId, (err, relatedInfo) => {
     if (err) {
       console.log('Error retrieving related info');
-      res.status(404).send();
+      res.status(404).send(err);
     } else {
       res.send(relatedInfo);
     }
   });
 });
 
+app.get('/cart', (req, res) => {
+  const { userToken } = req.query;
+  getCart(userToken, (err, response) => {
+    if (err) {
+      console.log('Error retrieving cart', userToken);
+      res.status(404).send(err);
+    } else {
+      res.send(response);
+    }
+  });
+});
+
+app.post('/cart', (req, res) => {
+  const params = {
+    usertoken: req.query.userToken,
+    sku_id: req.query.sku_id,
+    active: true,
+  };
+  addToCart(params, (err, response) => {
+    if (err) {
+      console.log('Error adding to cart');
+      res.status(404).send(err);
+    } else {
+      res.send(req.query.sku_id, 'added to cart');
```

Comment:
I think posts should have a 201 status code (default is 200). It is specified in the API documentation for FEC.